simtools/simtel/simulator_ray_tracing.py

```diff
@@ -184,7 +184,7 @@ def _make_run_command(
             command += super().get_config_option("mirror_align_random_distance", "0.")
             command += super().get_config_option("mirror_align_random_vertical", "0.,28.,0.,0.")
         command += " " + str(self._corsika_file)
-        command += " 2>&1 > " + str(self._log_file) + " 2>&1"
+        command += f" {self._log_file} 2>&1"
```

Comment:
Are you sure about this change? You won't be writing out to the log file this way.

Reply:
Good catch! Just tested this and it is not working here, although the same command works for the light emission package and `logfile 2>&1` is what is mentioned in the simtel_array docs.